data/json/items/armor.json

```diff
@@ -627,6 +627,28 @@
     "material_thickness": 5,
     "flags": [ "VARSIZE", "OUTER", "FANCY" ]
   },
+  {
+    "id": "armor_riot",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "riot armor",
+    "description": "Black armor used by riot police officers.  The padded layers and hard plastic shoulder guards offer good protection without much encumberance.  The word POLICE is emblazoned across the front.",
+	"looks_like": "kevlar",
+    "weight": 3000,
```

Comment:
It's only three kilograms?

Reply:
I've changed it to 4 kilograms, but according to the site below the full suit with arm and leg guards is 6,5 kilograms.

https://www.armorusa.com/product/riogard-riot-gear/